vpr/src/pack/prepack.cpp

```diff
@@ -1548,7 +1548,16 @@ static void update_chain_root_pins(t_pack_patterns* chain_pattern,
     for (const auto pin_ptr : chain_input_pins) {
         std::vector<t_pb_graph_pin*> connected_primitive_pins;
         get_all_connected_primitive_pins(pin_ptr, connected_primitive_pins);
+
+        /**
+         * It is required that the chain pins areconnected inside a complex
+         * block. Although it is allowed to have them disconnected in some of
```

Comment:
Looks good, thanks.
A couple of typos to fix:
areconnected (missing space)
"in some of modes of the block" -> "in some modes of the block"

Reply:
Ok, fixed that.